File: 058_length_of_last_word/Solution.go
package main

import "fmt"

func lengthOfLastWord(s string) int {
    length := 0

    for i := len(s) - 1; i >= 0; i-- {
        if s[i] == ' ' {
            if length > 0 {
                break
            } else {
                continue
            }
        }
        length++
    }

    return length
}

func main() {
    fmt.Println(lengthOfLastWord("Hello World "))
}

File: 058_length_of_last_word/Solution.java
import java.util.*;

class Solution {
    public int lengthOfLastWord(String s) {
        int sLen = s.length(), length = 0;

        for (int i = sLen - 1; i >= 0; i--) {
            if (s.charAt(i) == ' ') {
                if (length == 0) continue;
                else break;
            }
            length++;
        }

        return length;
    }

    public static void main(String[] args) {
        System.out.println(new Solution().lengthOfLastWord("Hello World "));
    }
}
